templates/cart/cart.php

```diff
@@ -109,6 +109,7 @@
 										'input_value' => $cart_item['quantity'],
 										'max_value'   => $_product->get_max_purchase_quantity(),
 										'min_value'   => '0',
+                                                                                'product_name' => $_product->get_name()        
```

Comment:
The indenting of this line looks to be incorrect. Please indent with tabs. If you use an editor which supports `.editorconfig` files it will help with this.